pages/index.tsx

```diff
@@ -1,17 +1,19 @@
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import SeoLayout from '../components/seo-layout/seo-layout.component';
+import styles from '../styles/Home.module.css';
 
 export default function Home() {
   return (
     <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
+      <SeoLayout
+        title={'Create Next App'}
```

Comment:
Keep all use facing strings in localised file.

Reply:
Rajat will add the localise file using in18

Reply:
Please add a todo in the code.

Reply:
Please add TODO in the code that needs to be localised.

Ex:

```
TODO agentraghav: Localize
```